src/main/resources/locales/en/delete-court-publication-confirmation.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think this should be 'publications'. Same on line 5 (and will be the same in the other local files)

Reply:
fixed.